src/server/clu_server_setup.c

```diff
@@ -0,0 +1,136 @@
+/* clu_server_setup.c
+ *
+ * Copyright (C) 2006-2021 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#include <wolfclu/clu_header_main.h>
+#include <wolfclu/clu_log.h>
+#include <wolfclu/clu_optargs.h>
+#include <wolfclu/server.h>
+
+#ifndef WOLFCLU_NO_FILESYSTEM
+
+static const struct option server_options[] = {
+    {"-port",           required_argument, 0, WOLFCLU_PORT                  },
+    {"-key" ,           required_argument, 0, WOLFCLU_KEYFILE               },
+    {"-cert",           required_argument, 0, WOLFCLU_CERTFILE              },
+    {"-clientcert",     required_argument, 0, WOLFCLU_CA                    },
+    {"-help",           no_argument,       0, WOLFCLU_HELP                  },
+    {"-h",              no_argument,       0, WOLFCLU_HELP                  },
+    {0,0,0,0}
+};
+
+static const char portFlag[]        = "-p";
+static const char keyFileFlag[]     = "-k";
+static const char certFileFlag[]    = "-c";
+static const char clientCertFlag[]  = "-A";
+
+static void wolfCLU_ServerHelp(void) 
+{
+    WOLFCLU_LOG(WOLFCLU_L0, "./wolfssl s_server");
+    WOLFCLU_LOG(WOLFCLU_L0, "\t-port <+int>");
+    WOLFCLU_LOG(WOLFCLU_L0, "\t-key <private key file name>");
+    WOLFCLU_LOG(WOLFCLU_L0, "\t-cert <cert file name>");
```

Comment:
need type (DER or PEM) if applicable. If both are handled then please list in the help print out that either DER or PEM can be used for key and cert file.

Reply:
Only PEM types are supported, as stated in help.